packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -196,9 +198,21 @@ export function SquadPageHeader({
           <PrivilegedMemberItem key={member.user.id} member={member} />
         ))}
         {privilegedLength > listMax && (
-          <PrivilegedMemberContainer className="h-fit font-bold text-text-tertiary typo-callout">
+          <button
+            type="button"
+            className={classNames(
+              'h-fit font-bold text-text-tertiary typo-callout',
+              memberContainerClasses,
+            )}
```

Comment:
This is supposed to be a custom button? 
Why is it not a guidelined one?

Reply:
From Figma, there is an override that happened. Do we proceed with the `Button` component while using `!` on its classes or do we go with a plain button?

![Screenshot 2024-05-27 at 8 34 38 PM](https://github.com/dailydotdev/apps/assets/13744167/7338a29b-13cb-41ea-9baa-ecfd4f662e11)

Reply:
Hmm maybe double check with design if they happy with this temp or they made some mistake?
(Because it feels like tertiary almost from the classes)

Reply:
Right. Let me send a message.